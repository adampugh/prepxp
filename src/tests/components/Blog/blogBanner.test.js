import React from "react";
import { shallow } from "enzyme";
import BlogBanner from "../../../components/Blog/blogBanner";

describe("<blogBanner />", () => {
    test("should match snapshot", () => {
        const wrapper = shallow(<BlogBanner />);
        expect(wrapper).toMatchSnapshot();
    });
});
