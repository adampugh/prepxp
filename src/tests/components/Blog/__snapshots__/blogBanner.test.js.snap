// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<blogBanner /> should match snapshot 1`] = `
<div
  className="blogBanner"
>
  <div
    className="container blogBanner__grid"
  >
    <div
      className="blogBanner__text"
    >
      <h1>
        Blog
      </h1>
      <p>
        Read articles from industry leaders and stay up to date with the latest trends in job recruitment.
      </p>
    </div>
    <div
      className="blogBanner__img"
    >
      <img
        alt=""
        src="interview2.png"
      />
    </div>
  </div>
  <div
    className="blogBanner__whiteEllipse"
  />
</div>
`;
